frontend/lib/src/controllers/add_account_controller.dart

```diff
@@ -12,4 +14,8 @@ class AddAccountController extends GetxController {
 
   bool get buttonEnabled => _buttonEnabled.value;
   set setButtonEnabled(bool newValue) => _buttonEnabled.value = newValue;
+
+  Future<Account?> addUserAccount(Account account) async {
```

Comment:
return whole response, FE can use the error response from BE if any field validation fails